client/gutenberg/extensions/presets/jetpack/utils/jetpack-logo.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { Path, Polygon, SVG } from '@wordpress/components';
+import classNames from 'classnames';
+
+const JetpackLogo = ( { size = 32, className } ) => (
+	<SVG
+		className={ classNames( 'jetpack-logo', className ) }
+		width={ size }
```

Comment:
Gutenberg will resize to 24 anyways, I believe.
```suggestion
		width={ 24 }
```